lib/build.js

```diff
@@ -191,7 +194,7 @@ class BuildModel extends BaseModel {
                             isPR: job.isPR(),
                             jobId: job.id,
                             pipelineId: pipeline.id
-                        })
+                        }, pipeline.scmContext)
```

Comment:
It is ok to  pass `scmContext` because `tokenGen` would take `scmContext` by this PR changes.
https://github.com/screwdriver-cd/screwdriver/pull/648